src/ProjectTemplates/test/GrpcTemplateTest.cs

```diff
@@ -35,6 +36,13 @@ public async Task GrpcTemplate()
             var buildResult = await Project.RunDotNetBuildAsync();
             Assert.True(0 == buildResult.ExitCode, ErrorMessages.GetFailedProcessMessage("build", Project, buildResult));
 
+            // These templates are HTTPS + HTTP/2 only which is not supported on Mac due to missing ALPN support.
```

Comment:
Shouldn't this be:
```c#
[ConditionalFact]
[OSSkipCondition(OperatingSystems.MacOSX)]
```
Instead? This pattern gives the impression that the test ran on Mac when it didn't do anything interesting. Either that or we should verify the behavior you get on a mac (I assume it gives a semi-useful error message?).

Reply:
Added a check for the error message.